source/welcome/whats-new-in-v72.rst

```diff
@@ -0,0 +1,183 @@
+
+Navigation is simplified, and now works similarly to channels. No more switching between workspaces - and the dashboard page has been removed. All the boards you're currently a member of from your current team will appear on the sidebar without needing to switch workspaces. Use the search box (Ctrl+K / Cmd+K) to find additional boards.
+
+.. image:: ../images/boards-whats-new-01.png
+  :alt: Boards sidebar categories can be customized for easier navigation.
+
+Boards can be organized under custom categories on a per-user level. So, users can organize their boards under categories that make sense to them without impacting the boards or categories for other users. Select the “...” options menu next to a custom category to manage your categories.
+
+.. image:: ../images/boards-whats-new-02.png
+  :alt: Select the context menu to create a new category, delete a category, or rename a category.
+
+Team sidebar
+~~~~~~~~~~~~
+
+Boards now supports a team sidebar so you can easily navigate between boards on different teams. If you’re a member of multiple teams, the team sidebar will appear on the left hand side. To switch teams, select any of the team icons from the team sidebar.
+
+.. image:: ../images/boards-whats-new-03.png
+  :alt: The team sidebar allows you to switch between teams from within boards.
+
+Linking boards to channels
+--------------------------
+
+The channel header’s board icon is now located on the channel `Apps Bar <https://docs.mattermost.com/configure/configuration-settings.html#enable-apps-bar>`_. When you select the **Boards** icon, you’ll open a new right-hand sidebar (RHS) where channel admins can search and link boards to the channel. To maintain the same organization, all the boards previously associated with the workspace will automatically appear on the RHS post-migration. Select a linked board to navigate directly to the board.
+
+.. image:: ../images/boards-whats-new-04.png
+  :alt: You can find the boards icon in the Apps Bar on the right-hand side of the screen.
+
+Channel members can search and link boards within the team where they are also board admins. Linking a board to a channel automatically grants all channel members Editor access to the board.
+
+.. note:: 
+  
+  A channel can be linked to multiple boards, but each individual board can only be linked to one channel at a time. Linking the same board to another channel will automatically remove the link from the previous channel.
+
+Channel members can also create a new board from the RHS. Doing so will automatically link the new board to the channel and grant channel members permissions to the board.
+
+.. image:: ../images/boards-whats-new-05.png
+  :alt: You can create a board from the RHS which is automatically added to the channel you're in.
+
+Sharing
+-------
+
+Select **Share** to view and edit permissions to a board.
+
+.. image:: ../images/boards-whats-new-06.png
+  :alt: Select the share button to open up the permissions options.
+
```

Comment:
Can we force a line break between these 2 images?

Reply:
Let me know if it looks better?